chapters/chap02-sec01.tex

```diff
@@ -5,8 +5,10 @@ \section{\zeth~Data Types}\label{zeth-protocol:zeth-data-types}
 We begin by describing, and giving intuition about, the data types (see~\cref{preliminaries:structured-data}) used in \zeth.
 We follow some design rationale from \zerocash~\cite{sasson2014zerocash}, and \zcash~\cite{zcashprotocol} in order to prevent the transaction malleability attack, and the Faerie Gold attack\cite[Section 8.4]{zcashprotocol}. We refer the reader to~\cref{appendix:trnm} for more details.
 
+In what follows \Curve~represents a curve with scalar field $\FFx{\rCURVE}$, satisfying the requirements of \cref{instantiation:zksnark}. The specification is described in terms of this generic curve, with examples and notes relating to specific instances of interest (namely \BNCurve~and \BLSCurve, see \cref{chap:instantiation}).
+
 \begin{description}
-    \item[\zethNoteDType] Represents a note in \zeth. It comprises the note's owner public address \apk, identifier $\rrho$, randomness \noter{}~and value \notev.
+    \item[\zethNoteDType] Represents a note in \zeth. Consists of the note owner's public address \apk, identifier $\rrho$, randomness \noter{}~and value \notev.
```

Comment:
I've combined the two sentences.  The previous sentence also has no subject (as these are contracted sentences explaining `\zethNoteDType`), so it flowed correctly, but a single sentence may be better. `It ...` was a bit unnatural, as it could refer to the notation rather than the note it represents.

Reply:
Ah right, I think `\zethNoteDType` was the implicit subject of the firs sentence, but that's right, strictly speaking `Represents` starts another sentence (hence the capital letter) which means that the subject is missing (same for other data types descriptions in this section). This didn't strike me at first glance, but we can rephrase.

> `It ...` was a bit unnatural, as it could refer to the notation rather than the note it represents.

What about `It...` -> `This data type consists of ...`?

Reply:
changed.